### PR TITLE
Stats #100

### PR DESCRIPTION
**Pull request by blefebvre**
Fix #100

Spec: https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?node-id=1461%3A12680&t=D8NcLzqwbsbEQeyF-0

Test URLs:
- Before: https://main--merative2--hlxsites.hlx.page/drafts/bruce/clinical-decision-support-dev
- After: https://issue-100-stats--merative2--hlxsites.hlx.page/drafts/bruce/clinical-decision-support-dev

Example doc: https://merative.sharepoint.com/:w:/r/sites/Merative.comWebsite/_layouts/15/Doc.aspx?sourcedoc=%7BE41AA492-2DA3-4640-9ACE-FB2206C0B7A4%7D&file=clinical-decision-support-dev.docx&action=default&mobileredirect=true&cid=51177fc8-8b5c-4c44-a023-77be88175dc3

Block details:

image

Mobile:

Screenshot 2023-02-06 at 11 40 16 AM

Tablet:

Screenshot 2023-02-06 at 11 39 58 AM

Desktop:

Screenshot 2023-02-06 at 11 39 39 AM

### CODE REVIEW

File: styles/lazy-styles.css [hunk 1]
```diff
@@ -136,4 +136,109 @@ main.error a.button-404  span{
...
+  align-items: start;
+  column-gap: var(--spacer-element-07);
+}
+
+/* Gradiant line, from purple to red */
+div.stat-wrapper .stat > div > div::before {
+  display: block;
+  content: " ";
+  height: 1px;
+  background-image: linear-gradient(270deg, #fa1e32, #90f);
```
Comment (arumsey) [inline styles/lazy-styles.css:168]: `var(--gradient-left-right)` should work here.

File: styles/lazy-styles.css [hunk 2]
```diff
@@ -136,4 +136,109 @@ main.error a.button-404  span{
...
+}
+
+div.stat-wrapper .stat h1 {
+  font-size: 96px;
+  line-height: 120%;
```
Comment (arumsey) [inline styles/lazy-styles.css:173]: `var(--line-height-120);`

File: styles/lazy-styles.css [hunk 3]
```diff
@@ -136,4 +136,109 @@ main.error a.button-404  span{
...
+  letter-spacing: 0;
+  font-weight: 300;
+  padding: 0;
+  margin: 0;
+}
+
+div.stat-wrapper .stat p {
+  font-size: var(--font-size-16);
+  letter-spacing: .01em;
```
Comment (arumsey) [inline styles/lazy-styles.css:182]: `var(--letter-spacing-001-em);`

File: styles/lazy-styles.css [hunk 4]
```diff
@@ -136,4 +136,109 @@ main.error a.button-404  span{
...
+    margin-left: unset;
+  }
+
+  div.stat-wrapper .stat {
+    padding: 80px var(--spacer-layout-036) var(--spacer-layout-03);
+  }
+}
+
+/* Desktop styles */
+@media (min-width: 1192px) {
```
Comment (arumsey) [inline styles/lazy-styles.css:236]: @amol-anand can confirm but I think `1200px` is the desktop breakpoint being used.

Comment (blefebvre) [reply styles/lazy-styles.css:236]: `1192` is used in the spec: https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?node-id=258%3A24&t=R9xGLS1DkoVO4xQu-0

image

**Review by amol-anand — APPROVED** [review]

File: styles/lazy-styles.css [hunk 5]
```diff
@@ -130,10 +130,115 @@ main.error a.button-404  span{
...
   font-weight: var(--font-weight-medium);
   white-space: nowrap;
   color: #90f;
   border-bottom: 2px solid #90f;
   line-height: 120%;
-}
\ No newline at end of file
+}
+
+/* Styles for Columns with the "stat" class, and "stat-wrapper" section metadata style */
```
[the remark this answers is not in the document]
Comment (blefebvre) [reply styles/lazy-styles.css]: I moved it all out of lazy-styles, only changes remaining were updates to use variables:

image

Were you looking at a previous commit?

File: styles/lazy-styles.css (hunk 4 above)
Comment (blefebvre) [reply styles/lazy-styles.css]: Understood, latest changes are using 1200: https://github.com/hlxsites/merative2/pull/14/files#diff-10e61bb0cdf350cc5fdc389b443350e3952faa739db6564793612e2ea72e391cR83

**Pull request merged**